linux.mk

```diff
@@ -173,7 +173,7 @@ export CONFIG_PATH ?= $(HOME)/config.yaml
 export MTU ?= 1500
 export MSS ?= 1500
 export PEER ?= server
-export TEST ?= udp_push_pop
+export TEST ?= udp-push-pop
```

Comment:
Does this mean we only run this one example test in the default case?

Reply:
It means that if someone simply invokes `make test-system-rust`, it will attempt to run the `udp-push-pop` example application, in `server` mode with `catnap` LibOS.

Specifically at this point I am just fixing the name of the default program. It is named `udp-push-pop` with dashes, not with underscores.

However, note that this is not the main fix that I aimed at in this PR.